lib/src/Base/Stat/IsotropicCovarianceModel.cxx

```diff
@@ -0,0 +1,235 @@
+//                                               -*- C++ -*-
+/**
+ *
+ *  Copyright 2005-2020 Airbus-EDF-IMACS-ONERA-Phimeca
+ *
+ *  This library is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public License
+ *  along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+#include "openturns/IsotropicCovarianceModel.hxx"
+#include "openturns/CovarianceModel.hxx"
+#include "openturns/CovarianceModelImplementation.hxx"
+
+BEGIN_NAMESPACE_OPENTURNS
+
+CLASSNAMEINIT(IsotropicCovarianceModel)
+
+
+/* Parameters constructor */
+IsotropicCovarianceModel::IsotropicCovarianceModel(const CovarianceModel & oneDimensional,
+  const UnsignedInteger inputDimension)
+  : CovarianceModelImplementation()
+{
+  setKernel(oneDimensional);
+  setInputDimension(inputDimension);
+  setActiveParameter(kernel_.getActiveParameter());
+  setOutputCorrelation(kernel_.getOutputCorrelation());
+  setScale(kernel_.getScale());
+  setAmplitude(kernel_.getAmplitude());
+  setNuggetFactor(kernel_.getNuggetFactor());
+}
```

Comment:
As kernel should be 1d (both for input & output), it is useless to work on correlation